authors/apps/articles/migrations/0001_initial.py

```diff
@@ -1,7 +1,9 @@
-# Generated by Django 2.1.3 on 2018-12-14 08:13
+# Generated by Django 2.1.3 on 2018-12-17 12:50
 
 from django.conf import settings
 from django.db import migrations, models
+import django.db.models.deletion
+import taggit.managers
```

Comment:
what does this import do/achieve?

Reply:
That is an autogenerated file created during migrations. The first importation is used when deleting models in the database while the second one helps in adding tags to the models.